scss/globals/top/_body.scss

```
@@ -218,4 +221,10 @@ body {
             // 重複していたメディアクエリの最小化
         }
     }
+}
+
+@font-face{
+    font-family: 'UDShingoKyokasho';
```

Stylelint (17.13.0)

[error] Expected no quotes around "UDShingoKyokasho" (font-family-name-quotes)

(font-family-name-quotes)